frontend/Dockerfile

```
@@ -1,18 +1,21 @@
 FROM node:16 as build
 
 WORKDIR /app
-COPY . .
-# disabled to speed up build, assuming built locally
-#RUN yarn
+COPY package.json .
+COPY yarn.lock .
+RUN yarn
```

for reproducible dependencies, easier to debug especially if used in CI:
```suggestion
RUN yarn --frozen-lockfile
```